### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by orikurt**
same as ori/secureRoutes

### CODE REVIEW

**Comment by aryehbeitz** [comment]
@orikurt 
no need to merge into master
you can deploy directly from develop
you can also set in heroku that auto-deploy will be from develop

File: config/locales/he.yml [hunk 1]
```diff
@@ -180,6 +180,7 @@ he:
       title: "חיפוש ערב זיכרון בסלון"
       subtitle: "כאן אפשר לחפש סלון פתוח בסביבת מגוריכם. <br> אם לא מצאת סלון להתארח בו, תמיד אפשר לארח בעצמך."
       zb_salons: "סלוני זיכרון בסלון"
+      no_salons: "לא מצאנו סלון מתאים. מומלץ לחפש גם ביישובים קרובים לשלך, או בתאריך אחר."
```
Comment (aryehbeitz) [inline config/locales/he.yml]: add the same string in the en.yaml file (yes, hebrew)

Comment (orikurt) [inline config/locales/he.yml]: you mean in english?

**Comment by aryehbeitz** [comment]
put the same hebrew text in the english file: en.yml

**Pull request merged**